### PR TITLE
unnecessary auth_brute_force

### PR DESCRIPTION
**Pull request by ernestotejeda**
https://github.com/OCA/server-auth/issues/38#issuecomment-462308921

Cc @Tecnativa

### CODE REVIEW

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 merged_modules = {
     # Odoo
+    'auth_brute_force': 'base',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py]: but `auth_brute_force` is not from `#Odoo`

please, put in a new `#OCA/server-auth` below like `OCA/web` case

Comment (ernestotejeda) [reply odoo/addons/openupgrade_records/lib/apriori.py]: thanks @mreficent

**Review by MiquelRForgeFlow — COMMENTED** [review]
Please, could you squash commits?

**Comment by pedrobaeza** [comment]
Not needed. We can squash on merge.

**Pull request merged**